cmd/telemetry-api/main.go

```diff
@@ -92,7 +87,7 @@ func main() {
 
 	authedHandler := limiter.AddRequestTimeout(
 		authMiddleware.CheckJWT(
-			auth.AddClaimHandler(server, &logger),
+			auth.AddClaimHandler(server, &logger, settings.VehicleNFTAddress, settings.ManufacturerNFTAddress),
```

Comment:
This is a bit gross.

Reply:
I'm including Manufacturer here to be illustrative and because we need it in #59. It's not being used here obviously.

Reply:
we could check for this in main-- if both of these are the same/ zero we definitely would have a big problem